app/assets/src/components/views/samples/samples_view.scss

```diff
@@ -63,7 +63,7 @@
   .separator {
     flex: 0 0 auto;
     height: 27px;
-    margin: 0 20px;
+    margin: 0 10px 0 20px;
```

Comment:
Why do you need to reduce this?

Reply:
After increasing the size of the icon to include the gray hover circle, the distance between the leftmost icon and the separator became too big.